ipapython/certdb.py

```diff
@@ -102,7 +102,7 @@ def find_cert_from_txt(cert, start=0):
     return (cert, e)
 
 
-def parse_trust_flags(trust_flags):
+def parse_trust_flags(trust_flags: str) -> TrustFlags:
```

Comment:
Just a question. We don't use the python typing hints anywhere else within the IPA core code. Should we have a project-wide discussion about it prior to including it?